tests/src/RunBenchmarks/benchmarks.xml

```diff
@@ -0,0 +1,669 @@
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+    </benchmark-suite>
+    <benchmark-suite>
+        <name>Fractals</name>
+            <benchmark>
+                <name>Fractals</name>
+                <directory>.\Fractals</directory>
+                <executable>FractalPerf.exe</executable>
+                <args>&gt; out</args>
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+    </benchmark-suite>
+    <benchmark-suite>
+        <name>Spec</name>
+            <benchmark>
+                <name>bzip2</name>
+                <directory>.\bzip2</directory>
+                <executable>bzip2.exe</executable>
+                <args>mscorlib.il &gt; out</args>
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+            <benchmark>
+                <name>Crafty</name>
+                <directory>.\crafty</directory>
+                <executable>crafty.exe</executable>
+                <args>&lt; crafty.in &gt; out</args>
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+    </benchmark-suite>
+    <benchmark-suite>
+        <name>Mono</name>
+            <benchmark>
+                <name>mono-binarytrees</name>
+                <directory>.\mono</directory>
+                <executable>binarytrees.csharp.exe</executable>
+                <args>20 &gt; out</args>
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+            <benchmark>
+                <name>mono-chameneos-redux</name>
+                <directory>.\mono</directory>
+                <executable>chameneosredux.csharp.exe</executable>
+                <args>6000000 &gt; out</args>
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+            <benchmark>
+                <name>mono-fannkuch-redux</name>
+                <directory>.\mono</directory>
+                <executable>fannkuchredux.csharp-3.exe</executable>
+                <args>12 &gt; out</args>
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+            <benchmark>
+                <name>mono-fasta</name>
+                <directory>.\mono</directory>
+                <executable>fasta.csharp-2.exe</executable>
+                <args>25000000 &gt; out</args>
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+            <benchmark>
+                <name>mono-fasta-redux</name>
+                <directory>.\mono</directory>
+                <executable>fastaredux.csharp.exe</executable>
+                <args>25000000 &gt; out</args>
+                <run-in-shell>true</run-in-shell>
+            </benchmark>
+            <benchmark>
+                <name>mono-knucleotide</name>
+                <directory>.\mono</directory>
+                <executable>knucleotide.csharp-4.exe</executable>
+                <args>0 &lt; knucleotide-input25000000.txt &gt; out</args>
```

Comment:
So stdin redirection is the only way to pass the input file? I wonder if my stdout comments apply here, too.